src/main/java/de/dataelementhub/rest/controller/v1/ElementController.java

```diff
@@ -384,4 +385,22 @@ public ResponseEntity updateMembers(@PathVariable(value = "urn") String urn,
       return new ResponseEntity<>(e, HttpStatus.BAD_REQUEST);
     }
   }
+
+  /**
+   * Get all available paths for a given element.
+   */
+  @GetMapping("/{urn}/paths")
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  public ResponseEntity getElementPaths(@PathVariable(value = "urn") String urn,
+      @RequestHeader(value = HttpHeaders.ACCEPT_LANGUAGE, required = false) String languages) {
+    try {
+      int userId = DataElementHubRestApplication.getCurrentUser().getId();
+      List<ElementPath> elementPaths = elementService.getElementPaths(userId, urn, languages);
+      return new ResponseEntity<>(elementPaths, HttpStatus.OK);
+    } catch (IllegalArgumentException e) {
+      return new ResponseEntity<>(e, HttpStatus.BAD_REQUEST);
+    } catch (IllegalStateException e) {
+      return new ResponseEntity<>(e.getMessage(), HttpStatus.UNPROCESSABLE_ENTITY);
```

Comment:
I do not think that 410 GONE is the right response in this case because the element still there and can be called using the get element endpoint. It is just OUTDATED.

In my opinion it is better to use 422 Unprocessable Entity because this way we are telling the user that our endpoint is understanding the content and the syntax is correct but we can not  process returning the elementPath because the specified element has an unacceptable status for this endpoint which is OUTDATED.